lib/cylc/config.py

```diff
@@ -709,6 +709,11 @@ def __init__(self, suite, fpath, template_vars=[], template_vars_file=None,
                     print >> sys.stderr, '  %s => %s' % e
                 raise SuiteConfigError('ERROR: cyclic dependence detected '
                                        '(graph the suite to see back-edges).')
+        elif graphing_disabled:
+            print >> sys.stderr, (
+                "WARNING: skipping cyclic dependence check"
+                "  (importing graphviz library failed)")
+
```

Comment:
The cyclic dependency check is only done during validation (not, e.g. when parsing the suite prior to a run), so the logic needs to be something like this:
```python
if self.validation:
   if graphing_disabled:
         print >> sys.stderr, "WARNING..."
   else:
         # Cyclic check...
```